### PR TITLE
[chunk] add function is_in_fused

### PR DESCRIPTION
**Pull request by 1SAA**

### CODE REVIEW

File: elixir/chunk/fetcher.py [hunk 1]
```diff
@@ -64,6 +64,10 @@ def get_one_chunk(self, tensor: torch.Tensor) -> Chunk:
     def get_chunks(self, tensors: list[torch.Tensor]) -> list[Chunk]:
         return self.group.tensors_to_chunks(tensors)
 
+    def is_in_fused(self, tensor: torch.Tensor):
+        chunk = self.get_one_chunk(tensor)
+        return chunk.rcache_fused
```
Comment (FrankLeeeee) [inline elixir/chunk/fetcher.py:69]: what is rcache?

Comment (1SAA) [reply elixir/chunk/fetcher.py:69]: Memory Pool

Comment (FrankLeeeee) [reply elixir/chunk/fetcher.py:69]: I mean what does `r` stand for?

**Review by FrankLeeeee — APPROVED** [review]
Please add doc for the name `rcache` as it can be quite misleading.

**Pull request merged**